zerver/views/users.py

```diff
@@ -529,7 +531,6 @@ def add_bot_backend(
         (default_events_register_stream, ignored_sub) = access_stream_by_name(
             user_profile, default_events_register_stream_name
         )
-
```

Comment:
We generally don't change anything unrelated to the issue at hand.

Reply:
Removed the change